packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,9 +1,75 @@
-# Change Log
+# Changelog
 
-All notable changes to the "teamsfx-toolkit" extension will be documented in this file.
+## 2.0.0 - May 25 2021
+2.0.0 is a major version upgrade for Teams Toolkit with many new improvements and supports.
+- Add: new design of UI and command palette list
+- Add: new getting started guide, samples and doc link from toolkit
+- Add: new sign-in experience for M365 and Azure
+- Add: new interactive flow for screating new app
```

Comment:
should it be "creating"?

Reply:
followed some other CHnagelog, but I can use the Docker extension format. "Added""Fixed""Enhanced", fixing now.

Reply:
No, I mean "screating" should be "creating"

Reply:
this has been fixed. and made some more formating